packages/loot-core/src/server/admin/app.ts

```
@@ -0,0 +1,255 @@
+      return { error: err.reason };
+    }
+  }
+
+  return null;
+});
+
+app.method('access-delete-all', async function ({ fileId, ids }) {
+  const userToken = await asyncStorage.getItem('user-token');
+  if (userToken) {
+    try {
+      debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)